ci-operator/step-registry/optional-operators/cvp-common/aws-osd/optional-operators-cvp-common-aws-osd-workflow.yaml

```diff
@@ -0,0 +1,34 @@
+workflow:
+  as: optional-operators-cvp-common-aws-osd
+  steps:
+    cluster_profile: aws-osd-msp
+    env:
+      #BASE_DOMAIN: cpaas-ci.devcluster.openshift.com
```

Comment:
Just a question - how is the base domain set for the MSP OSD cluster profile?

Reply:
I would like to know also.

Reply:
Hi @dirgim 
we have a way to configure a https://docs.openshift.com/dedicated/applications/deployments/osd-config-custom-domains-applications.html 
i think that should help. 
I am still figuring out if the custom domains work, and where to get the certificates from.